python-checks/src/test/resources/checks/afterJumpStatement.py

```diff
@@ -2,15 +2,17 @@
     break
 
 for i in range(1):
-    break; print(i)
+    break; print(i) # Noncompliant {{Remove the code after this "break".}}
+#   ^^^^^
 
 for i in range(1):
-    continue
+    continue # Noncompliant {{Remove the code after this "continue".}}
     print(i)
 
 if True:
     print(1)
-    raise TypeError("message")
+    raise TypeError("message") # Noncompliant {{Remove the code after this "raise".}}
```

Comment:
@vilchik-elena With the current issue message, isn't it a bit misleading to highlight only the keyword? Could someone understand that `return x` should be changed to `return`?